clover.rb

```diff
@@ -99,6 +99,17 @@ def assets
     if error[:code] == 204
       # nothing
     elsif api? || runtime? || request.headers["Accept"] == "application/json"
+      # :nocov:
+      if Config.test?
+        case e
+        when Sequel::ValidationFailed, DependencyError, Validation::ValidationFailed, CloverError, Authorization::Unauthorized
```

Comment:
`DependencyError`, `Validation::ValidationFailed`, and `Authorization::Unauthorized` are all subclasses of `CloverError`. Does the `when` keyword match with the parent class?